### PR TITLE
Adding Voxel Segmentation Workflow documentation

### PR DESCRIPTION
**Pull request by subeeshvasu**

### CODE REVIEW

File: documentation/VoxelSegmentation/VoxelSegmentation.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+- After annotating a few voxels, switch the Live update on (using the Live Update button). 
+This trains an initial classifier and results in a first prediction. 
+The imperfect results of this prediction can be inspected by the options shown at the bottom left panel. 
+Training and label update follows similar steps to the ones mentioned in [Pixel Classification Workflow](https://www.ilastik.org/documentation/pixelclassification/pixelclassification.html). 
+The differences come in the following aspects. 
+After each live update, some regions in the segmentation outputs will be highlighted with topUncertainty. 
+These are the regions that are most uncertain as per the classifier decision, and therefore should be given more emphasis in the next step of labeling. 
+By clicking on “suggest slice,” the user can access the slices with maximum uncertainty. 
+In each of these slices, regions with maximum uncertainty will also be highlighted.
```
Comment (k-dominik) [inline documentation/VoxelSegmentation/VoxelSegmentation.md]:
```suggestion
```

File: documentation/VoxelSegmentation/VoxelSegmentation.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# Voxel Segmentation Workflow
```
Comment (k-dominik) [inline documentation/VoxelSegmentation/VoxelSegmentation.md]:
```suggestion
---
title: Voxel Segmentation workflow
tagline: Voxel Segmentation with Active Learning
category: "Documentation"
group: "workflow-documentation"
weight: 1
---
# Voxel Segmentation Workflow
```

Need this boilerplate in order to make the docs show up in the toc

File: documentation/VoxelSegmentation/VoxelSegmentation.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+Let’s go through the controls of this applet from top to bottom:
+
+- n_segments: Number of supervoxels that the input volume will be divided into. 
+Higher value results in finer partitioning of the input volume.
+
+- Compactness: Control the compactness of a supervoxel. 
+A higher value emphasizes more spatial proximity resulting in more compact clusters.
+
+- max_iter: Number of iterations used to compute the supervoxel clusters. 
+The algorithm typically converges in 4 to 10 iterations.
```
[the remark this answers is not in the document]
Comment (subeeshvasu) [reply documentation/VoxelSegmentation/VoxelSegmentation.md]: Typically, the difference in the output with respect to the number of the iterations is minor after 10 iterations. So, keeping the iterations as 10 can be declared as a default standard. To check the convergence, user can increase the iterations until the successive variations in the output becomes negligible.

[the remark this answers is not in the document]
Comment (k-dominik) [reply documentation/VoxelSegmentation/VoxelSegmentation.md]: ok thank you!

**Comment by k-dominik** [comment]
Hi Subeesh, I resized the images and made some fancy links to them so they resize properly. I'll also squash a few commits before merging (that means I'll force push to your master if you don't have reservations against that) but other than that I think we're good.

**Comment by subeeshvasu** [comment]
HI Dominik,

Sounds great. Please go ahead.

**Pull request merged**